### ISSUE TITLE
Tree view on `prs list` command instead of plain list

### ISSUE THREAD

**Issue by ivan-avalos** [timvisee/prs#2]
Hello! I think it would be better if `prs list` behaved similarly to `pass ls`, showing passwords in a tree instead of a plain list, like this:

```
├── password1
├── folder
│   ├── password2
│   └── password3
└── password4
```

**Comment by timvisee** [comment]
Good idea, that looks better.

Would showing a tree view through `ls` by default be the best option? Creating a different subcommand or flag for it would be possible as well.

**Comment by timvisee** [comment]
I've decided to change the default behavior of `ls` to show the tree view as you've suggested. Will be implementing that now.

**Comment by timvisee** [comment]
I've implemented this in a7ca38389454042b29bf729356683ede5cf86f3a, and released it as part of `v0.2.3`.

This version should be available as soon as [this](https://gitlab.com/timvisee/prs/-/pipelines/265598002) release pipeline succeeds.
